src/Utils/Strings.php

```diff
@@ -332,10 +332,10 @@ public static function compare($left, $right, $len = NULL)
 	 * @param  string|array
 	 * @return string
 	 */
-	public static function findPrefix($strings)
+	public static function findPrefix(...$strings)
 	{
-		if (!is_array($strings)) {
-			$strings = func_get_args();
+		if (is_array($strings[0])) {
+			$strings = $strings[0];
```

Comment:
This function is used very rarely, but common usecase is probably `findPrefix($a, $b)`, so `...$strings` fits better and posibility to use typehint `string ...$strings` sound good.